.travis.yml

```diff
@@ -11,8 +10,6 @@ os:
   - linux
   - osx
 
-osx_image: xcode61
```

Comment:
I think we can use different version of Xcode. By removing this, I just wanted to use the default setting (no need in dgtal).